src/test/java/com/epam/tc/hw1/BaseTest.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.tc.hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+public class BaseTest {
+
+    public Calculator calculator = new Calculator();
+
+    @BeforeTest
```

Comment:
It will be better to use `@BeforeMethod` instead of `@BeforeTest`.
`@BeforeTest` does not work before each method which marked `@Test` annotation

Reply:
To be honest, I tried to use @BeforeMethod and @AfterMethod but in this case I caught NullPointerException and could not resolve this problem another way, so I used another annotations.